crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -95,6 +95,19 @@ impl DrawState {
     ) -> bool {
         self.index_buffer == Some((buffer, offset, index_format))
     }
+
+    /// Resets tracking state
+    pub fn reset_tracking(&mut self) {
+        self.pipeline = None;
+        self.bind_groups.iter_mut().for_each(|val| {
```

Comment:
I'm not sure I understand why you set the values to None instead of just clearing the entire vec so it's emtpy.

Reply:
See previous comment: https://github.com/bevyengine/bevy/pull/14948/files#r1733639227